src/resources/ChateauMantegna.ts

```diff
@@ -0,0 +1,19 @@
+import { runCombat, visitUrl } from "kolmafia";
+import { get } from "../property";
+
+export function have(): boolean {
+  return get("chateauAvailable");
+}
+
+export function paintingMonster(): Monster | null {
+  return get("chateauMonster") ?? null;
```

Comment:
Does this not return `null` by default? I tried to make the `get` code do that for monster keys